torch/linalg/__init__.py

```diff
@@ -2012,7 +2011,8 @@
     out (Tensor, optional): output tensor. Ignored if `None`. Default: `None`.
 
 Returns:
-    A real-valued tensor, even when :attr:`A` is complex.
+    A real-valued tensor, even when :attr:`A` is complex. If :attr:`p` is one of `('fro', 'nuc', inf, -inf, 1, -1)`, 
+    and the :attr:`A` matrix is not invertible, returns `inf`.
```

Comment:
No need to be so specific here.

Reply:
I guess based off https://pytorch.org/docs/stable/notes/numerical_accuracy.html#extremal-values-in-linalg, the original `Returns` doc section should be left as it was.